### PR TITLE
added support for Amazon linux 2018 (using sysv not systemd)

### PR DESCRIPTION
**Pull request by cpepe**
#### Pull Request (PR) description
Adds support for Amazon linux 2018. The default behavior attempts to use systemd and fails.

#### This Pull Request (PR) fixes the following issues
There is no issue associated with this PR

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @cpepe, can you please rebase against our latest master branch?

File: manifests/params.pp [hunk 1]
```diff
@@ -16,8 +16,13 @@
...
         $service_lockfile      = '/var/lock/subsys/confluence'
         $refresh_systemd       = false
+      } elsif $facts['os']['name'] == 'Amazon' and $facts['os']['release']['major'] == '2018' {
+        $service_file_location = '/etc/init.d/confluence'
+        $service_file_template = 'confluence/confluence.initscript.erb'
+        $service_lockfile      = '/var/lock/subsys/confluence'
+        $refresh_systemd       = false
       } else {
-        fail("Only osfamily ${facts['os']['family']} 6 and 7 and supported")
+        fail("Only osfamily ${facts['os']['family']} 6 and 7, and Amazon Linux 2018 are supported")
```
Comment (bastelfreak) [inline manifests/params.pp]: can you add the version to the metadata.json?

Comment (cpepe) [reply manifests/params.pp]: I'll review. It looks like the logic has shifted to checking for systemd/sysv first. In that case this PR may be unnecessary. Let me get some testing and deeper dive.

**Comment by cpepe** [comment]
We are reviewing and may be able to close this PR without merging or recommend a different change. I'll have something soon.

File: manifests/config.pp [hunk 2]
```diff
@@ -5,6 +5,7 @@
 class confluence::config(
   $tomcat_port         = $confluence::tomcat_port,
   $tomcat_max_threads  = $confluence::tomcat_max_threads,
+  $tomcat_max_http_header_size = $confluence::tomcat_max_http_header_size,
```
Comment (bastelfreak) [inline manifests/config.pp:8]: please add a datatype for the new parameter

File: manifests/init.pp [hunk 3]
```diff
@@ -11,6 +11,9 @@
   $jvm_permgen                                                   = '256m',
   $java_opts                                                     = '',
   Variant[String,Array[String]] $catalina_opts                   = '',
+  $printgcdetails                                                = '-XX:-PrintGCDetails',
```
Comment (bastelfreak) [inline manifests/init.pp:14]: Please add datatypes for the new parameters

**Comment by SimonHoenscheid** [comment]
Amazon Linux 2018 is EOL. Closed

**Pull request closed without merging**